src/types/label.js

```diff
@@ -196,25 +202,28 @@ function getCalloutSideAdjust(position, options) {
   return side;
 }
 
-function resolveCalloutPosition(element, options) {
+function resolveCalloutPosition(element, options, rotation) {
```

Comment:
Same here, rename element to properties

Reply:
yes, everywhere the "element" variable is used.

Reply:
done